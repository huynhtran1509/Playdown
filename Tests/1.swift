println("Playdown")

//: Convert a Swift playground to Markdown
//: Perfect for blog posts
var i = 0
i++

/*:
# Use Markdown in your comments!
Check the documentation at Apple's website
*/
println(i)
