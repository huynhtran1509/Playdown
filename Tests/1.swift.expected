```swift
println("Playdown")
```

Convert a Swift playground to Markdown
Perfect for blog posts

```swift
var i = 0
i++
```

# Use Markdown in your comments!
Check the documentation at Apple's website

```swift
println(i)
```
